index.html

```diff
@@ -220,6 +220,66 @@ <h3>Multi-Post Stories</h3>
             </div>
           </section>
 
+          <!--Contact-form-->
+          <section id="contact" class="c-form">
+            <p class="con-us">CONTACT US</p>
+            <p class="con-par">
+              If you have an application you are
+              interested in developing, a feature that you
+              need built or a project that needs coding.
+              I'd love to help with it.
+            </p>
+            <form action="https://formspree.io/f/xqknvevr" method="post">
+              <ul class="un-list">
+                <li class="li-item">
+                  <label for="name">
+                    <input type="text" name="name" id="name" class="name-input detail-input" placeholder="Enter your names..."/>
+                  </label>
+                </li>
+                <li class="li-item">
+                  <label for="name">
+                    <input type="email" name="email" id="email" class="email-input detail-input" placeholder="example@gmail.com"/>
+                  </label>
+                </li>
+                <li class="li-item">
+                  <label for="textarea">
+                    <textarea name="text" id="text"  class="text-input detail-input" placeholder="Enter text here..." cols="23" rows="5"></textarea>
+                  </label>
+                </li>
+                <li class="btn-container">
+                  <button type="submit" class="submit-btn">Submit</button>
+                </li>
+              </ul>
+            </form>
+
+            <hr>
+
+          </section>
+
+          <footer class="footer">
+            <nav>
+              <ul class="media-icons">
+                <li>
+                  <a href="#" class="fa fa-github"></a>
+                </li>
+                <li>
+                  <a href="#" class="fa fa-linkedin"></a>
+                </li>
+                <li>
+                  <a href="#" class="fa fa-twitter"></a>
+                </li>
+                <li>
+                  <a href="#" class="fa fa-medium"></a>
+                </li>
+                <li>
+                  <a href="#" class="fa fa-facebook"></a>
+                </li>
+              </ul>
+            </nav>
+          </footer>
+
+          <a class="email-me" href="mailto:tembofrank600@gmail.com?subject=Reaching20%Out">tembofrank600@gmail.com</a>
```

Comment:
- [ ] Kindly make sure your `footer` design sticks to the one from the Figma(e.g: colors used on its content, spacing top && bottom (margins or padding),...). YOu can check every detail on the Figma template to stick your design to it.

| Figma Design | Your Design |
| ------------------- | ------------------- |
| ![Screenshot from 2022-07-06 01-36-42](https://user-images.githubusercontent.com/83042742/177433921-3d027613-d145-4b95-a265-f8f871807bb6.png) | ![Screenshot from 2022-07-06 01-33-21](https://user-images.githubusercontent.com/83042742/177433703-2e44c768-dd04-4e72-99e3-75d63ee0e41c.png) |